src/database/model/ride.ts

```diff
@@ -0,0 +1,10 @@
+import {Model} from '@nozbe/watermelondb';
+import {field, date} from '@nozbe/watermelondb/decorators';
+
+export default class Ride extends Model {
+  static table = 'ride';
```

Comment:
shouldn't this be plural `rides`?

Reply:
I've changed `sensors` -> `sensor` so you'll need to clear your app's db as the migrations won't be compatible.